src/modules/endpoint/DatabaseConnector.js

```diff
@@ -385,6 +437,32 @@ class DatabaseConnector {
     await this.writeFile(path, JSON.stringify(json))
   }
 
+  /**
+   * Deletes the resource caches of all but the latest used cities
+   * @return {Promise<void>}
+   */
+  async deleteOldResourceCaches (context: DatabaseContext) {
+    if (!context.cityCode) {
+      throw Error('cityCode mustn\'t be null')
+    }
+    // $FlowFixMe flow does not get we're filtering for null
+    const lastUsages = await this.loadLastUsages()
+    const cachesToDelete = lastUsages.filter(it => it.lastUsage !== null)
+      .filter(it => it.city !== context.cityCode)
+      // Sort last usages chronological, from oldest to newest
+      .sort((a, b) =>
+        a.lastUsage.isBefore(b.lastUsage) ? -1 : (a.lastUsage.isSame(b.lastUsage) ? 0 : 1)
+      )
+      .slice(0, -(MAX_RESOURCE_CACHES - 1))
```

Comment:
This should be:
```suggestion
      .slice(0, -MAX_RESOURCE_CACHES)
```

Reply:
No, it shouldn't.  We are excluding the current city, too.

Reply:
Ahh you're right. Can you add a comment?

Reply:
Sure